### DISCUSSION TITLE
Manual exchange rates being ignored by Storefront API

### DISCUSSION THREAD

**Discussion by zamson** [Shopify/storefront-api-feedback#187] [Help]
I'm trying to use manual exchange rates on my Shopify store as described in [this documentation](https://help.shopify.com/en/manual/markets/pricing/exchange-rates#manual-exchange-rate-conversions). 

However, when I use a GraphQL query to retrieve product information, the query does not reflect the manual exchange rates that I have set up and instead returns the automatic conversion rates. Nevertheless, when I add the product to the cart, the correct price is displayed in the cart and checkout.

Here's an example of the GraphQL query I'm using to retrieve product information by providing a product handle and country code:

```
query getProductByHandle($handle: String!, $country: CountryCode!) @inContext(country: $country) {
  product(handle: $handle) {
    title
    variants(first: 100) {
      nodes {
        id
        price {
          amount
          currencyCode
        }
      }
    }
  }
}

# variables
{
  "handle": "delicate-pearl-necklace",
  "country": "US"
}
```

When I set a fixed price on the product level, the query returns the correct fixed price, indicating that the issue is specifically related to the manual exchange rate settings or percentage marked up rates.

Any help appreciated.

**Comment by johncraigcole** [comment]
Hey @zamson this should work as you expect. I just ran some tests where I set a manual conversion rate on some of the secondary markets in my store and the SFAPI product query with `@inContext(country:AU)` is returning the correct price where I set a 1.25 conversion rate for the Australian market. That being said, there could be configuration differences between your store and mine. 

It will difficult to troubleshot this issue here as we don't have proper authentication on the Shopify store. If you reach out to Shopify Support that will get you authenticated and allow them to review your store setup.
